java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/suites/NightlyFunctionalSystemTestSuite.java

```diff
@@ -21,6 +21,6 @@
 import sleeper.systemtest.suite.testutil.SystemTestSuite;
 
 @SystemTestSuite
-@ExcludeTags("expensive")
+@ExcludeTags({"expensive1", "expensive2", "expensive3"})
```

Comment:
We could avoid stating all the numbers here if we give the expensive annotations an "expensive" tag as well as the numbered tags.